lib/features/auth/cubit/auth_cubit.dart

```diff
@@ -200,14 +200,13 @@ class AuthCubit extends Cubit<AuthState> {
     ///TODO: I discussed this with @MaikelStuivenberg and will leave it as is for now. Until we will redesign the auth flow
     await _authRepositoy.logout();
 
-    AnalyticsHelper.setFamilyAppTracking(isOn: false);
-
     emit(
       state.copyWith(
         status: AuthStatus.unauthenticated,
         email: state.user.email,
       ),
     );
+    unawaited(AnalyticsHelper.clearUserProperties());
```

Comment:
If we make a new method. Isn't it better to do the unawaited part in there and not make the method async?
I guess there is never a reason to wait, right?

Reply:
Good catch, fully agreed!